test/message/message-processorSpec.ts

```diff
@@ -90,4 +90,18 @@ describe('the message processor only forwards valid, authorized messages', () =>
 
     expect(lastAuthenticatedMessage).toBe(message as any)
   })
+
+  it('does not process auth requests', () => {
```

Comment:
this test doesn't make sense, processing a authentication request once should not result in an error message.